src/primaite/agents/aegis/gllm.py

```diff
@@ -113,6 +181,16 @@ def train_loop(model: GLLM, dataloader: DataLoader, network_desc: str, n_epochs:
 
             model.optimizer.step()
 
-        print(f"Epoch {epoch} Loss:", loss.item(), gllm_responses[0].replace("\n", "//n"))
-
+        if epoch % checkpoint_every == 0:
```

Comment:
Loving the checkpointing very nice 👍

Reply:
If you actually ran it you'd know that there was a bug <3

Reply:
💀